modules/png-extraction/ImageExtractor.py

```diff
@@ -170,101 +191,80 @@ def extract_images(filedata, i, png_destination, flattened_to_level, failed, is1
     found_err=None
     filemapping = ""
     fail_path = ""
-    try:
-        im = ds.pixel_array # pull image from read dicom
-        imName=os.path.split(filedata.iloc[i].loc['file'])[1][:-4] # get file name ex: IM-0107-0022
-
-        if flattened_to_level == 'patient':
-            ID = filedata.iloc[i].loc['PatientID']  # Unique identifier for the Patient.
-            folderName = hashlib.sha224(ID.encode('utf-8')).hexdigest()
-            # check for existence of patient folder. Create if it does not exist.
-            os.makedirs(png_destination + folderName,exist_ok=True)
-        elif flattened_to_level == 'study':
-            ID1 = filedata.iloc[i].loc['PatientID']  # Unique identifier for the Patient.
-            try:
-                ID2 = filedata.iloc[i].loc['StudyInstanceUID']  # Unique identifier for the Study.
-            except:
-                ID2='ALL-STUDIES'
-            folderName = hashlib.sha224(ID1.encode('utf-8')).hexdigest() + "/" + \
-                         hashlib.sha224(ID2.encode('utf-8')).hexdigest()
-            # check for existence of the folder tree patient/study/series. Create if it does not exist.
-            os.makedirs(png_destination + folderName,exist_ok=True)
-        else:
-            ID1=filedata.iloc[i].loc['PatientID']  # Unique identifier for the Patient.
-            try:
-                ID2=filedata.iloc[i].loc['StudyInstanceUID']  # Unique identifier for the Study.
-                ID3=filedata.iloc[i].loc['SeriesInstanceUID']  # Unique identifier of the Series.
-            except:
-                ID2='ALL-STUDIES'
-                ID3='ALL-SERIES'
-            folderName = hashlib.sha224(ID1.encode('utf-8')).hexdigest() + "/" + \
-                         hashlib.sha224(ID2.encode('utf-8')).hexdigest() + "/" + \
-                         hashlib.sha224(ID3.encode('utf-8')).hexdigest()
+    im = ds.pixel_array # pull image from read dicom
```

Comment:
Some of these lines do not have the actual change. Make sure only your changes are in the pull request.

This is a best practice for any open-source development and not just Emory BMI. (It helps with tracking who did the changes and also makes it easy to review).